app/static/js/slcManagement.js

```diff
@@ -25,8 +25,50 @@ $(document).ready(function() {
       }
     }
   });
-});
+  const emailSelectedButton = $("#emailSelectedButton");
+  const selectAllOthersButton = $("#selectAllOthersButton");
+  const instructorCheckboxes = $(".instructorCheckbox");
+
+  emailSelectedButton.on("click", function () {
+    const selectedEmails = Array.from(instructorCheckboxes)
+      .filter((checkbox) => checkbox.checked)
+      .map((checkbox) => checkbox.getAttribute("data-email"))
+      .join(";");
+
+    if (selectedEmails) {
+      const windowRef = window.open(`mailto:${selectedEmails}?subject=Renew Course Proposal`, '_blank');
+  
+      windowRef.focus();
+    
+      setTimeout(function(){
+        if(!windowRef.document) {
+            windowRef.close();
+        }
+      }, 500);
+    } else {
+      alert("Please select at least one instructor to email.");
+    }
+  });
 
+  selectAllOthersButton.on("click", function () {
+    var myButton = document.getElementById('selectAllOthersButton');
```

Comment:
this isn't 1845 why are you using var the only time people do is if they're stealing code from a 2007 stack overflow post

Reply:
The real comment here is that you shouldn't use `document.getElementById`, use the jquery methods